Extension/package.json

```diff
@@ -981,6 +981,12 @@
           "default": "Enabled",
           "description": "%c_cpp.configuration.codeFolding.description%",
           "scope": "window"
+        },
+        "C_Cpp.supportExtensionVscodeVersion": {
```

Comment:
We do not want a setting for this.  It should just be read from the cpptools.json file.